File: A_Vlad_and_the_Best_of_Five.cpp
#include <iostream>
#include <string>
#include <unordered_map>

int main() {
    int t;
    std::cin >> t;  // Read the number of test cases

    while (t--) {
        std::string str;
        std::cin >> str;  // Read each test case

        std::unordered_map<char, int> charCount;
        for (char c : str) {
            charCount[c]++;
        }

        if (charCount['A'] > charCount['B']) {
            std::cout << "A" << std::endl;
        } else {
            std::cout << "B" << std::endl;
        }
    }

    return 0;
}

File: A_Vlad_and_the_Best_of_Five.py
t = int(input())  # Read the number of test cases

for _ in range(t):
    string = input()  # Read each test case and add it to the list

    map = {"A": 0, "B": 0}
    for char in string:
        map[char] += 1
    
    if map["A"] > map["B"]:
        print("A")
    else:
        print("B")